ProcessLib/HeatTransportBHE/BHE/BHE_1U.h

```diff
@@ -0,0 +1,213 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Eigen>
+
+#include "BaseLib/Error.h"
+
+#include "BHECommon.h"
+#include "FlowAndTemperatureControl.h"
+#include "PipeConfiguration1U.h"
+
+namespace ProcessLib
+{
+namespace HeatTransportBHE
+{
+namespace BHE
+{
+class BHE_1U final : public BHECommon
+{
+public:
+    BHE_1U(BoreholeGeometry const& borehole,
+           RefrigerantProperties const& refrigerant,
+           GroutParameters const& grout,
+           FlowAndTemperatureControl const& flowAndTemperatureControl,
+           PipeConfiguration1U const& pipes)
+        : BHECommon{borehole, refrigerant, grout, flowAndTemperatureControl},
+          _pipes(pipes)
+    {
```

Comment:
Why aren't the implementations of the constructor and some other methods of this class in the cpp file?

Reply:
As suggested by TF, the implementation of the constructor and other functions have been moved to the cpp file. The one still staying in the h file is the "assembleRMatrices" function. This function is called in the constructor of local BHE element assembler to initialize the thermal resistance matrices.